app/views/admin/adventures/new.html.erb

```diff
@@ -5,5 +5,5 @@
 
   <%= render('form', adventure: @adventure) %>
   <hr>
-  <%= link_to(t('.all'), admin_adventures_path, class: 'btn secondary') %>
+  <%= link_to(t('.all'), admin_adventures_path(introduction: @introduction), class: 'btn secondary') %>
```

Comment:
Why?

Reply:
I didn't quite get it to work, but I had a vision of being able to choose introduction at the 'Administrate/introductions'-page and after that not encountering adventures from other introductions when editing or creating adventures, as I can see how this could be confusing since you've actually selected an introduction. If you wish to see all adventures there is a separate link for 'administrate/adventures'. What do you think?